operations/workplace/people/hr-cadences.md

```diff
@@ -10,15 +10,74 @@ Listening sessions are 1x1 meeting with your HR partner. The intent of the HR Li
 
 Feedback themes will help drive HR priorities, communication and/or clarification efforts and even coaching opportunities for managers and leaders.
 
-**Cadence** Your HR partner will schedule time with you via Google Calendar. The goal is to meet with every person at least once per quarter. This may change due to HR team bandwidth. However, every staff member is encouraged to schedule time with someone on HR if there is not an existing scheduled meeting or if there’s an urgent topic.
+**Cadence** 
+The goal is monthly for managers and quarterly for non-managers. Your HR partner will schedule time with you via Google Calendar. This may change due to HR team bandwidth. However, every staff member is encouraged to schedule time with someone on HR if there is not an existing scheduled meeting or if there’s an urgent topic.
```

Comment:
```suggestion
The goal is monthly for managers and quarterly for non-managers. Your HR partner will schedule time with you via Google Calendar. This may change due to HR team bandwidth. However, every staff member is encouraged to schedule time with someone on HR even if there isn't an existing scheduled meeting or if there’s an urgent topic.
```